crypto/cmp/cmp_ctx.c

```diff
@@ -459,6 +461,34 @@ int OSSL_CMP_CTX_set1_secretValue(OSSL_CMP_CTX *ctx, const unsigned char *sec,
     return 1;
 }
 
+/* Returns the cert chain computed by OSSL_CMP_certConf_cb(), NULL on error */
+STACK_OF(X509) *OSSL_CMP_CTX_get1_newChain(const OSSL_CMP_CTX *ctx)
+{
+    if (ctx == NULL) {
+        CMPerr(0, CMP_R_NULL_ARGUMENT);
+        return NULL;
+    }
+    if (ctx->newChain == NULL)
+        return sk_X509_new_null();
+    return X509_chain_up_ref(ctx->newChain);
+}
+
+/*
+ * Copies any given stack of inbound X509 certificates to newChain
+ * of the OSSL_CMP_CTX structure so that they may be retrieved later.
+ */
+int ossl_cmp_ctx_set1_newChain(OSSL_CMP_CTX *ctx, STACK_OF(X509) *newChain)
```

Comment:
I'm failing to see where this gets called with anything other than NULL for its last param?

Reply:
Right, in this PR it is not really used, and this is an artifact of my mentioned recent split of PRs.
It is going to be used after merging in #12741, which currently contains in `crypto/cmp/cmp_client.c:`
```
    /* TODO(3.0) add when available: (void)ossl_cmp_ctx_set1_newChain(ctx, chain); */   
```